### PR TITLE
Feat multiple queue listener

### PR DESCRIPTION
**Pull request by lsh-0**
observer needs to handle more than article updates, so it's update listener has been made more generic and the article-specific logic shifted into ingest_logic

### CODE REVIEW

File: src/observer/management/commands/update_listener.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import sys, json
```
Comment (giorgiosironi) [inline src/observer/management/commands/update_listener.py]: I'd keep an eye on this listener modules to make them extractable into a `bus-sdk-python` similar to https://github.com/elifesciences/bus-sdk-php as we have seen them pop up at least in `observer`, `elife-metrics`, `bot-lax-adaptor`. There is some machinery around the catching of exceptions, deletion of used messages and seamless New Relic monitoring that could be shared by projects.

Comment (seanwiseman) [reply src/observer/management/commands/update_listener.py]: or add them to https://github.com/elifesciences/bus-sdk-python

Comment (giorgiosironi) [reply src/observer/management/commands/update_listener.py]: That's the perfect place!

Comment (lsh-0) [reply src/observer/management/commands/update_listener.py]: I'll open tickets to add that functionality and integration into the python projects.

**Comment by lsh-0** [comment]
thanks for the feedback guys, I'll merge after the press-packages/profiles work is merged in as this PR will require some updates. would rather update this than that

**Pull request merged**